### ISSUE TITLE
Cannot access repos of wsl

### ISSUE THREAD

**Issue by dor6-deepcred** [desktop/desktop#14365]
### The problem

when opening a repo from wsl, i get a message "repo is potentially unsafe".
when I click `Trust repository...` nothing happens. the only other option is `remove`.

### Release version

2.9.14

### Operating system

windows 11

### Steps to reproduce the behavior

1. go to a repo from wsl
2. click `Trust repository...`
3. wait for nothing to happen

### Log files

[2022-04-14.desktop.production.log](https://github.com/desktop/desktop/files/8488628/2022-04-14.desktop.production.log)

### Screenshots

![image](https://user-images.githubusercontent.com/90688282/163378547-dada6c64-92f1-49fe-9090-3c10843d3290.png)

**Comment by sanderderks** [comment]
I have the exact same problem after updating to the latest version of GitHub Desktop just now.

**Comment by FrameMuse** [comment]
I have the same problem.
Here's my logs of this error when clicking
```
2022-04-14T09:57:43.021Z - error: [ui] `git remote -v` exited with an unexpected code: 128.
stderr:
error: failed to get owner for '//wsl.localhost/Ubuntu/home/kotto/github/react-template' (1)
fatal: unsafe repository ('//wsl.localhost/Ubuntu/home/kotto/github/react-template' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory //wsl.localhost/Ubuntu/home/kotto/github/react-template

2022-04-14T09:57:43.068Z - info: [ui] launching: 2.9.14 (Windows 10.0.22000)
2022-04-14T09:57:43.076Z - info: [ui] execPath: 'C:\Users\framemuse\AppData\Local\GitHubDesktop\app-2.9.14\GitHubDesktop.exe'
2022-04-14T09:57:43.100Z - error: [ui] `git remote -v` exited with an unexpected code: 128.
stderr:
error: failed to get owner for '//wsl.localhost/Ubuntu/home/kotto/github/react-template' (1)
fatal: unsafe repository ('//wsl.localhost/Ubuntu/home/kotto/github/react-template' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory //wsl.localhost/Ubuntu/home/kotto/github/react-template

2022-04-14T09:57:43.265Z - error: [ui] `git remote -v` exited with an unexpected code: 128.
stderr:
error: failed to get owner for '//wsl.localhost/Ubuntu/home/kotto/github/react-template' (1)
fatal: unsafe repository ('//wsl.localhost/Ubuntu/home/kotto/github/react-template' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory //wsl.localhost/Ubuntu/home/kotto/github/react-template
```

**Comment by FrameMuse** [comment]
Also may be related to https://github.com/desktop/desktop/issues/14336

**Comment by michalkortas** [comment]
I have the same problem on WSL2.

**Comment by N58** [comment]
I have the same problem on WSL2 after updating GitHub Desktop. Repository was already added and window with "Trust repository..." button appeared. Pressing it doesn't do anything. I deleted repository, tried to add it back and I've got another window
![image](https://user-images.githubusercontent.com/21145217/163386724-09b95996-741a-4fb5-a117-4598618e116a.png)
Pressing "add an exception for this directory" also doesn't do anything.

**Comment by niik** [comment]
This issue is being tracked in https://github.com/desktop/desktop/issues/14336. Please subscribe to that to follow along. The [latest update](https://github.com/desktop/desktop/issues/14336#issuecomment-1098490973) as of writing this is that WSL and NAS (or path beginning with `//`) will not work. This is due to a security fix in Git for Windows and we're working with that team to come up with a permanent fix. In the meantime there are links to the installer for an older version of Desktop here: https://github.com/desktop/desktop/issues/14336#issuecomment-1098309322.

**Comment by lbp003** [comment]
Have the same issue after updating

**Comment by Ibra3264** [comment]
Same issue

**Comment by marjunmakinano** [comment]
I'm having the same issue

**Comment by notaspecial** [comment]
same here

**Comment by Danimaxpd** [comment]
Same Here :S

**Comment by lbp003** [comment]
They have fixed it in Version 2.9.15-beta1
[Release note](https://desktop.github.com/release-notes/?env=beta)
For now, I'm using it.
